test/test_environment.py

```diff
@@ -13,18 +14,21 @@
 from asv import util
 
 
+WIN = (os.name == "nt")
+
+
 try:
     util.which('python2.7')
     HAS_PYTHON_27 = True
-except RuntimeError:
-    HAS_PYTHON_27 = False
+except (RuntimeError, IOError):
+    HAS_PYTHON_27 = (sys.version_info[:2] == (2, 7))
```

Comment:
Yes, but if the present Python version is 2.7, then 2.7 is available even if `python2.7` is not in PATH. This is the situation when installing Python from Python.org on Windows.